src/System-BasicCommandLineHandler/BasicCommandLineHandler.class.st

```diff
@@ -77,6 +77,8 @@ BasicCommandLineHandler class >> startUp: isImageStarting [
 	"only handle when lauching a new image"
 	isImageStarting ifFalse: [ ^ self ].
 
+	JS ifConnected: [ ^self ].
```

Comment:
You should implement different "default" command line handler for JS environment. (if special startup logic is really required).

Reply:
Agreed. The intent here is just to decline starting this facility if the host platform is Web. There should be some delegation to the host platform here.